tests/components/plex/test_config_flow.py

```diff
@@ -457,9 +459,65 @@ def init_config_flow(hass):
 
 
 async def test_option_flow(hass):
-    """Test config flow selection of one of two bridges."""
+    """Test config options flow selection."""
 
-    entry = MockConfigEntry(domain=config_flow.DOMAIN, data={}, options=DEFAULT_OPTIONS)
+    mock_plex_server = MockPlexServer(load_users=False)
+
+    MOCK_SERVER_ID = MOCK_SERVERS[0][config_flow.CONF_SERVER_IDENTIFIER]
+    hass.data[config_flow.DOMAIN] = {
```

Comment:
Here were mocking integration code. It would be more robust to just patch the library server and set up the integration so we don't need to mock our own code.

Reply:
Agree, but there are currently no tests which actually set up the integration. I'd like to do that in the future.